app/assets/stylesheets/nucore-overrides.scss

```diff
@@ -9,3 +9,21 @@ Example of this to include here:
   $headerBackgroundColor: #571963;
   $headerColor: white;
 */
+
+// Shared
+$darkGrey: #3F3B26;
+$grey: #9B9B9B;
+$lightGrey: #D4D1D1;
+$lighterGrey: #F3F3F3;
+
+// Manage Mode
+$orange: #FF7126;
```

Comment:
Looks like the use/manage variables aren't used yet. Just want to make sure on that.

Reply:
This is correct. Currently they are being used on our use/manage CSS branch, but we want to minimize the redundancy and therefore think it is best to include them here. Assuming this branch is the first of our design changes to get merged, it is the best candidate for this commit.